20151222/rails-5.0.0.beta1-demo/db/schema.rb

```diff
@@ -11,7 +11,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151222140949) do
+ActiveRecord::Schema.define(version: 20151223221159) do
+
+  create_table "comments", force: :cascade do |t|
+    t.integer  "post_id"
+    t.text     "body"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.